4-Deployment/README.md

```diff
@@ -0,0 +1,341 @@
+        - Keep **State** as **Enabled**.
+        - Select the **Add scope** button on the bottom to save this scope.
+1. In the app's registration screen, select the **API permissions** blade in the left to open the page where we add access to the APIs that your application needs.
+   - Select the **Add a permission** button and then,
+     - Ensure that the **Microsoft APIs** tab is selected.
+     - In the *Commonly used Microsoft APIs* section, select **Microsoft Graph**
+     - In the **Delegated permissions** section, select the **User.Read** in the list. Use the search box if necessary.
+     - Select the **Add permissions** button at the bottom.
+   - Select the **Add a permission** button and then,
+     - Ensure that the **My APIs** tab is selected.
+     - In the list of APIs, select the API `msal-react-spa`.
+     - In the **Delegated permissions** section, select the **Access 'msal-react-api'** in the list. Use the search box if necessary.
+     - Select the **Add permissions** button at the bottom.
+
+#### Configure the app (msal-react-spa) to use your app registration
+
+Open the project in your IDE (like Visual Studio or Visual Studio Code) to configure the code.
+
+> In the steps below, "ClientID" is the same as "Application ID" or "AppId".
+
+1. Open the `App/src/authConfig.js` file.
+1. Find the key `Enter_the_Application_Id_Here` and replace the existing value with the application ID (clientId) of `msal-react-spa` app copied from the Azure portal.
+1. Find the key `Enter_the_Tenant_Info_Here` and replace the existing value with your Azure AD tenant ID.
+1. Find the key `Enter_the_Redirect_Uri_Here` and replace the existing value with the Redirect URI for `msal-react-spa`. (by default `http://localhost:3000/`).
+1. Find the key `Enter_the_Web_Api_Uri_Here` and replace the existing value with the URI of your function API (by default `/api/hello`).
+1. Find the key `Enter_the_Web_Api_Scope_Here` and replace the existing value with APP ID URI that you've registered earlier, e.g. `api://****-****-********-********/access_as_user`
+
+## Deployment
+
+There are basically **3** stages that you will have to go through in order to deploy your projects and enable authentication:
+
+1. Create a repository on GitHub and commit your project.
+1. Create a **Static Web App** via VS Code extensions.
+1. Update **Azure AD** **App Registration** with deployed website URI that you have just obtained.
+
+### Deploy the app (msal-react-spa)
+
+There are various ways to deploy your applications to **Azure Static Web Apps**. Here we provide steps for deployment via **VS Code Azure Static Web Apps Extension**.
+
+> We recommend reading the [Quickstart: Building your first static site with Azure Static Web Apps](https://docs.microsoft.com/azure/static-web-apps/getting-started?tabs=vanilla-javascript) for gaining familiarity with the process outlined below.
+
+#### Step 1: Deploy the app
```

Comment:
```suggestion
#### Step 1: Deploy the app
```
Step 1 is to create a new repo as per the list just above, maybe we can add a blurb here to put these step numbers back into phase?